test/grammars/jstestfuzz-test.ne

```diff
@@ -0,0 +1,15 @@
+@preprocessor jstestfuzz
+
+@{%
+const moo = require("moo");
```

Comment:
Any chance you can use the custom tokenizer functions here instead of the moo lexer? No worries if not. I just thought  it could look more like real-world jstestfuzz after TIG-2073